packages/drone-simulator/src/config.js

```diff
@@ -2,17 +2,11 @@ const config = require('@iteam/config')({
   file: `${__dirname}/../config.json`,
   defaults: {
     port: 3030,
-    osrm: 'http://localhost:5000',
-    redis: {
-      host: '127.0.0.1',
-      port: 6379,
-    },
-    elasticSearch: 'http://localhost:9200',
+    osrm: process.env.OSRM_HOST || 'http://localhost:5000',
```

Comment:
Name this `osrmHost` or put the env-variable under`OSRM` maybe?

Reply:
For sure. n1!